### PR TITLE
Issue#55: Style Home Page

### LINKED ISSUE #55: Design the HomePage for authenticated and unauthenticated views

### PR DESCRIPTION
**Pull request by zahrafalak**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

Designed the HomePage views for Unauthenticated and Authenticated audience referencing the [teams' figma design](https://www.figma.com/design/v0vHRdn5Gq8kNJSepE5xM8/TCL--GrocerEase?node-id=0-1&node-type=canvas&t=whiQgh3967Oy6e9O-0). Adjusted some of the default styling coming in from React Bootstrap elements to ensure they align with our design requirements. 

## Related Issue

Closes #55 

## Acceptance Criteria

- [ ]  Align the design as close to the [teams' figma design](https://www.figma.com/design/v0vHRdn5Gq8kNJSepE5xM8/TCL--GrocerEase?node-id=0-1&node-type=canvas&t=whiQgh3967Oy6e9O-0) as possible
- [ ]  The styles are responsive across mobile and tablet screens. 

## Type of Changes

`design` `UI updates`

### Before

Screenshot 2024-10-14 at 3 25 46 PM

Screenshot 2024-10-14 at 3 27 01 PM

### After

Unauthenticated HomePage:
Screenshot 2024-10-14 at 3 18 53 PM

Screenshot 2024-10-14 at 3 19 25 PM

Authenticated HomePage:
Screenshot 2024-10-14 at 3 32 46 PM

Screenshot 2024-10-14 at 3 33 04 PM

## Testing Steps / QA Criteria

1. Visit the [preview URL](https://tcl-77-smart-shopping-list--pr65-fz-design-homepage-hzadu9e3.web.app/) for this PR. 
2. In your browser dev tools, inspect the styling at 425px screen width for mobile, and 768px for tablet for unauthenticated and authenticated HomePage. 
3. Check the functionality of all buttons and interactive elements to ensure that nothing is broken.

### CODE REVIEW

**Review by RossaMania — APPROVED** [review]
I LOVE this! The pages look so nice and organized, and I love the grid look for buttons!
I LOVE how organized the classes are!

File: src/views/authenticated/AuthenticatedHome.tsx [hunk 1]
```diff
@@ -10,14 +11,12 @@ interface Props {
...
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
-
+		<div className="home">
 			{user && (
 				<>
-					<ul>
+					<CreateList user={user} setListPath={setListPath} />
+					<h3 className="heading-text">Your Lists</h3>
```
Comment (bbland1) [inline src/views/authenticated/AuthenticatedHome.tsx:18]: Nice add to have a way to show the user these are their lists!

File: src/views/unauthenticated/UnauthenticatedHome.tsx [hunk 2]
```diff
@@ -7,30 +8,34 @@ export function UnauthenticatedHome() {
...
-				<p>New to GrocerEase:</p>
-				<Button
-					onClick={() => navigate("/about")}
-					aria-label="Navigate to the about application page."
-					className="m-2"
-				>
-					Learn More
-				</Button>
-				<SignInButton isSignIn={false} />
+				<h3 className="heading-text">Welcome Back:</h3>
```
Comment (bbland1) [inline src/views/unauthenticated/UnauthenticatedHome.tsx:22]: I love the seperation of the buttons here and switching the learn more to the below the sign back in button!

File: src/styles/_mixins.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// ====== MEDIA ========
+@mixin tablet {
```
Comment (bbland1) [inline src/styles/_mixins.scss:2]: so these are mixins how cool!

**Review by bbland1 — APPROVED** [review]

**Pull request merged**